config.md

```diff
@@ -1,13 +1,12 @@
 # <a name="containerConfigurationFile" />Container Configuration file
 
-The container's top-level directory MUST contain a configuration file called `config.json`.
```

Comment:
> Did you mean to drop this line?

Yes, see “Also drop the `config.json` existence MUST because” and later.  If that is contentious, I can add it back in this PR and file a follow-up PR for its removal.